docs/data/toolpad/core/components/account/account.md

```diff
@@ -30,16 +30,57 @@ When signed out, the component renders as an inline sign in button within the da
 
 ## Customization
 
-### Slot Props
-
-The underlying `signInButton`, `signOutButton` and `iconButton` components can be customized by passing in `slotProps` to the `Account` component.
+### Locale Text
 
 Labels for the sign in and sign out buttons can be customized through the `localeText` prop.
 
+{{"demo": "AccountLocale.js", "bg": "outlined" }}
+
+### Slot Props
+
+The underlying `signInButton`, `signOutButton`, `iconButton` and `userDetailsContainer` sections can be customized by passing in `slotProps` to the `Account` component.
+
 {{"demo": "AccountCustom.js", "bg": "outlined" }}
 
 ### Slots
 
-You can pass in your own items to the `Account` menu through the `menuItems` slot to add additional menu items in the space between the user's account details and the sign out button, to create larger, more complex menus:
+You can pass in your own components to completely override the default components inside the `Account` popover through the `slots` prop.
+
+#### Content
+
+Use the `content` slot to add any additional options in the space between the user's account details and the sign out button, to create larger, more complex menus:
+
+##### Account Switcher
+
+{{"demo": "AccountSlotsAccountSwitcher.js", "bg": "gradient"}}
+
+The `content` prop can take any React component, so you can use it to display information instead of adding menu items:
+
+##### Crypto Wallet
+
+{{"demo": "AccountSlotsInfo.js", "bg": "outlined" }}
```

Comment:
This feels a bit like an unrealistic demo. Can't we just improve the account switcher instead, make it nice looking with avatar and create button, not that menu?

Reply:
Agreed on improving the account switcher to make it nicer looking; at the same time I think we can keep this just for information purposes given it mirrors multiple crypto UX patterns